arbeitszeit_flask/templates/company/create_cooperation.html

```diff
@@ -11,12 +11,7 @@ <h1 class="title">
                 <p>
                     {{ gettext("Here you can create a new cooperation.") }}
                     <br><br>
-                    {{ gettext("A cooperation contains similar products. The prices for all products in a cooperation
-                    are
-                    averaged to reduce price competition. All producers still receive the credits on sale corresponding
-                    to
-                    how they planned production. This means that producers receive the same amount of credit to their
-                    account whether they are part of a cooperation or not.") }}
+                    {{ gettext("A cooperation contains similar products.  The prices for all products in a cooperation are averaged to reduce price competition.  All producers still receive the credits on sale corresponding to how they planned production. This means that producers receive the same amount of credit to their account whether they are part of a cooperation or not.") }}
```

Comment:
This was necessary since the reformatting did change the message ID. The in turn disconnected the message from its translation.